README.md

```diff
@@ -36,9 +36,12 @@ $ go test -v test/*.go
 
 # Currently supported crypto primitives
 
-The crypto primitives and schemes (primitives combined in some more complex protocol) supported by Emmy are listed in the table below.
+The crypto primitives and schemes (schemes are primitives combined in some more complex protocol) supported by emmy are listed in the table below.
+ZKP primitives and schemes are collected in crypto/zkp/primitives and crypto/zkp/schemes respectively.
+Each of the ZKP primitives contains a function (at the beginning of the file) which demonstrates how proof should be executed.
+Client-server communication via gRPC which enable execution of ZKPs over the internet is in client and server folders.
```

Comment:
Can you please use backticks around package names? `crypto/zkp/primitives`, `crypto/zkp/schemes`. Also, please use "package" instead of "folder", e.g. correct L42 to:

 "Client-server communication via gRPC which enables execution of ZKPs over the internet is in `client` and `server` packages".